cmd/start.go

```diff
@@ -42,9 +42,14 @@ var startServerCmd = &cobra.Command{
 	Use:   "server",
 	Short: "Start the server",
 	Run: func(cmd *cobra.Command, args []string) {
+		var err error
 		config.Config.IsServer = isServer
 		config.Config.Nodes = make(map[string]config.Node)
-		config.Config.TLSParams = server.TLSBuildServerConf()
+		config.Config.TLSParams.TLSConfig, err = config.TLSBuildServerConf()
+		if err != nil {
+			log.Println(colors.Red("Unable to load TLS configuration. Shutting down."))
+			gracefulShutdown()
```

Comment:
why gratefulShutdown here ? That function wait for ctrl c to quit.

Reply:
replaced with an exit